.github/workflows/cache-node-modules.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    branches:
+      - develop
+      - 'epic/**'
+      - 'release/**'
+name: Cache node modules
+
+env:
+  NODE_VERSION: '16'
+
+jobs:
+  cacheNodeModules:   
+    name: Cache root node_modules
```

Comment:
nitpick... think it can be 'cache node modules'? since we cache cypress ones that isnt on the root level?

Reply:
It would be 'root' of cypress :P